src/Stryker.Core/Stryker.Core/Reporters/DashboardReporter.cs

```diff
@@ -91,5 +57,37 @@ public void OnStartMutantTestRun(IEnumerable<IReadOnlyMutant> mutantsToBeTested,
         {
             // Method to implement the interface
         }
+
+        private async Task UploadBaseline(JsonReport mutationReport)
+        {
+            var projectVersion = _options.CurrentBranchCanonicalName ?? _options.FallbackVersion;
+
+            var reportUrl = await _dashboardClient.PublishReport(mutationReport.ToJson(), projectVersion);
+
+            if (reportUrl != null)
+            {
+                _logger.LogDebug($"\nYour baseline stryker report has been uploaded to: \n {reportUrl} \nYou can open it in your browser of choice.");
+            }
+            else
+            {
+                _logger.LogDebug("Uploading to stryker dashboard failed...");
+            }
+        }
+
+        private async Task UploadHumanReadableReport(JsonReport mutationReport)
+        {
+            var reportUrl = await _dashboardClient.PublishReport(mutationReport.ToJson(), _options.ProjectVersion);
+
+            if (reportUrl != null)
+            {
+                _chalk.Green($"\nYour stryker report has been uploaded to: \n {reportUrl} \nYou can open it in your browser of choice.");
+            }
+            else
+            {
+                _chalk.Red("Uploading to stryker dashboard failed...");
```

Comment:
This also needs to be logged to the logger, I don't think chalk gets put in the logs.